templates/terraform/custom_delete/active_directory_domain_trust.go.erb

```diff
@@ -0,0 +1,72 @@
+    config := meta.(*Config)
```

Comment:
Hm, what is different from the generated code here?  A comment might help make it clearer why this is being done.

Reply:
The generate DELETE method isn't including the {trust: } object in the response body thus custom_delete is needed

Reply:
Hm.  I wish we could solve that in a more targeted way.  Let me check with Dana and see if we're okay to do it this way or if we need a more complicated (but easier to maintain) change.

Reply:
Okay, this is fine!  Can you make a big noisy comment in api.yaml and terraform.yaml that says that this resource's delete function needs to be modified any time we edit the schema?  After that, as long as the tests are passing and all seems to be working, I'm good to approve.